pkg/ingress/model_builder.go

```diff
@@ -40,7 +40,8 @@ func NewDefaultModelBuilder(k8sClient client.Client, eventRecorder record.EventR
 	authConfigBuilder AuthConfigBuilder, enhancedBackendBuilder EnhancedBackendBuilder,
 	trackingProvider tracking.Provider, elbv2TaggingManager elbv2deploy.TaggingManager,
 	vpcID string, clusterName string, defaultTags map[string]string, externalManagedTags []string, defaultSSLPolicy string,
-	backendSGProvider networkingpkg.BackendSGProvider, enableBackendSG bool, disableRestrictedSGRules bool, logger logr.Logger) *defaultModelBuilder {
+	backendSGProvider networkingpkg.BackendSGProvider, enableBackendSG bool, disableRestrictedSGRules bool, disableAWSVpcCNIflag bool,
+	logger logr.Logger) *defaultModelBuilder {
```

Comment:
I don't see the change in [the ingress group reconcile](https://github.com/openshift/aws-load-balancer-controller/blob/main/controllers/ingress/group_controller.go#L55) which is supposed to pass the boolean from the feature gate.

Reply:
Fixed, had missed that commit.